experimental/cloudstrapper/playbooks/roles/agw-infra/files/cfnMagmaAgwPublic.json

```diff
@@ -0,0 +1,55 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Parameters": {
+      "paramSubnetSgi": {
+        "Type": "String",
+	"Description": "SGi Subnet",
+	"Default": "subnet-0884f3b19999bf3ad"

```

Comment:
These are default values that can be set by the user if doing repeated tests on the same Orchestrator deployment. I will look at removing defaults altogether so that they get picked up dynamically for every run (the code already does that)

Reply:
The CloudFormation files are actually picked from the roles/cfn directory. Removing the "files" directory completely.